pkg/controller/controller.go

```diff
@@ -23,6 +25,11 @@ import (
 	"github.com/yahoo/k8s-athenz-istio-auth/pkg/zms"
 )
 
+const (
+	queueNumRetries = 3
+	queueKey        = "sync"
```

Comment:
is this queueKey temporary?

Reply:
Yes, this will be replaced with the athenz domain key once https://github.com/yahoo/k8s-athenz-istio-auth/pull/14 is merged.